# Generated by Django 5.1.2 on 2024-10-20 18:20

from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('products', '0006_alter_product_unique_together_product_is_returnable_and_more'),
        ('reviews', '0003_productreview_updated_at_alter_productreview_rating_and_more'),
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.RemoveConstraint(
            model_name='productreview',
            name='unique_review',
        ),
        migrations.AddConstraint(
            model_name='productreview',
            constraint=models.UniqueConstraint(fields=('product', 'customer'), name='unique_review', violation_error_message='You have already reviewed this product.'),
        ),
    ]
